packages/react-components/src/components/chart/utils/getXAxis.ts

```diff
@@ -19,6 +19,6 @@ export const getXAxis = (viewportInMs: ViewportInMs, axis?: ChartAxisOptions): X
     },
     splitNumber: 6,
     min: 0,
-    max: viewportInMs.isDurationViewport ? viewportInMs.end : undefined,
+    max: viewportInMs.isDurationViewport ? viewportInMs.end : Date.now(),
```

Comment:
Date.now will not change the x-axis right ? 

this is basically 

`max: Date.now()`